appveyor.yml

```diff
@@ -22,7 +22,7 @@ install:
   - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
 
   # Install the dependencies of the project (skip nibabel for speed)
-  - "conda install --yes --quiet setuptools numpy scipy matplotlib scikit-learn nose mayavi pandas h5py PIL patsy"
+  - "conda install --yes --quiet setuptools numpy scipy matplotlib scikit-learn nose mayavi pandas h5py PIL patsy pyside"
```

Comment:
pyside is a qt implementation that traits uses as backend. Without it the tests produce an error when the GIU-classes are initialized because traitsui notices that there is no backend that implements all the required functionality.